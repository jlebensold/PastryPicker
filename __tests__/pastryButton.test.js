// @flow
import React from 'react';
import PastryButton from '../pastryButton';
import renderer from 'react-test-renderer';
/*
import Enzyme, { shallow } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16'
Enzyme.configure({adapter: new Adapter()})
*/

test('renders isActive', () => {
  const tree = renderer.create(
    <PastryButton onPress={(t) => { }} label={'Croissant'} isActive={true} />
  ).toJSON();
  expect(tree).toMatchSnapshot();
});
/*
test('renders isActive = false', () => {
	const button = shallow(
		<PastryButton onPress={ (t) => {} } label="MyLabel" isActive={false} />);
	console.log(button.find('TouchableHighlight'));
});
*/
