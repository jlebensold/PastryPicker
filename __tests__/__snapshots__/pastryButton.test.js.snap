// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders isActive 1`] = `
<View
  style={
    Object {
      "margin": 10,
    }
  }
>
  <View
    accessibilityComponentType={undefined}
    accessibilityLabel={undefined}
    accessibilityTraits={undefined}
    accessible={true}
    hasTVPreferredFocus={undefined}
    hitSlop={undefined}
    isTVSelectable={true}
    nativeID={undefined}
    onLayout={undefined}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Array [
        Object {
          "backgroundColor": "transparent",
        },
        Array [
          Object {
            "backgroundColor": "#5A8282",
            "borderRadius": 10,
            "justifyContent": "center",
            "minWidth": 140,
            "padding": 10,
          },
          Object {
            "backgroundColor": "#CD7734",
          },
        ],
      ]
    }
    testID={undefined}
    tvParallaxProperties={undefined}
  >
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
      style={
        Object {
          "color": "#FFF",
          "fontSize": 18,
        }
      }
    >
      Croissant
    </Text>
  </View>
</View>
`;
